src/arith/analyzer.cc

```diff
@@ -115,13 +115,18 @@ bool Analyzer::CanProve(const PrimExpr& expr) {
   return false;
 }
 
+// see other .cc file
```

Comment:
Which other cc file?

Reply:
This was just an accidental commit, I fixed this, applied a change from a branch with Max by accident.